docs/topics/Voice_Connections.md

```diff
@@ -266,8 +266,19 @@ If successful, the Voice server will respond with an [Opcode 9 Resumed](#DOCS_TO
 }
 ```
 
-If the resume is unsuccessful—for example, due to an invalid session—the websocket connection will close with the appropriate [close event code](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/voice-close-event-codes). You should then follow the [Connecting](#DOCS_RESOURCES_VOICE_CONNECTS/connecting-to-voice) flow to reconnect.
+If the resume is unsuccessful—for example, due to an invalid session—the WebSocket connection will close with the appropriate [close event code](#DOCS_TOPICS_OPCODES_AND_STATUS_CODES/voice-close-event-codes). You should then follow the [Connecting](#DOCS_TOPICS_VOICE_CONNECTS/connecting-to-voice) flow to reconnect.
 
 #### IP Discovery
 
-Generally routers on the internet mask or obfuscate UDP ports through a process called NAT. Most users who implement voice will want to utilize IP discovery to find their external IP and port which will then be used for receiving voice communications. To retrieve your external IP and port, send a 70-byte packet with empty data past the 4-byte ssrc. The server will respond back with another 70-byte packet, this time with a NULL-terminated string of the IP, with the port encoded in a **little endian** unsigned short stored in the last two bytes of the packet.
+Generally routers on the Internet mask or obfuscate UDP ports through a process called NAT. Most users who implement voice will want to utilize IP discovery to find their external IP and port which will then be used for receiving voice communications. To retrieve your external IP and port, send the following UDP packet to your voice port (all numeric fields are big endian):
+
+| Field           | Description                                                    | Size     |
+| --------------- | ------------------------i------------------------------------- | -------- |
+| Type            | Values 0x1 and 0x2 indicate request and response, respectively | 2 bytes  |
+| Length          | Message length excluding Type and Length fields (value 70)     | 2 bytes  |
+| SSRC            | Unsigned integer                                               | 4 bytes  |
+| Address         | Null-terminated string in response                             | 64 bytes |
+| Port            | Unsigned short                                                 | 2 bytes  |
+
+
+Earlier and now deprecated version of this message was 70-byte long including `SSRC`, `Address` and `Port` fields. The request should include four-byte SSRC (big endian). The server responds back with another 70-byte packet, four-byte SSRC (little endian) followed with a NULL-terminated string of the IP, with the port encoded in a **big endian** unsigned short stored in the last two bytes of the packet.
```

Comment:
Why is the SSRC in the response little endian? Earlier you specify "all numeric fields are big endian" and I think the SSRC is a numeric field. Or am I reading this wrong?

Reply:
I will clarify it. It has historic reasons in the old message. I meant the big endian comment to the new message.

Reply:
Text updated.

Reply:
Oh it wasn't clear to me that the entire paragraph there is about the old version

Reply:
Yeah, could we put like a "Deprecated Behavior" subheading on this paragraph to make that distinction clear?

Reply:
Header added.